src/main/java/de/siphalor/amecs/impl/ModifierPrefixTextProvider.java

```diff
@@ -42,15 +33,35 @@ public BaseText getText(Variation variation) {
 		return text;
 	}
 
-	public enum Variation {
-		COMPRESSED(null), TINY(COMPRESSED), SHORT(TINY), NORMAL(SHORT);
+	public static enum Variation {
+		COMPRESSED(".tiny"),
+		TINY(".tiny"),
+		SHORT(".short"),
+		NORMAL("");
 
-		public static Variation WIDEST = NORMAL;
+		public static final Variation WIDEST = NORMAL;
+		public static final Variation SMALLEST = COMPRESSED;
 
-		public final Variation shorter;
+		public final String translateKeySuffix;
 
-		Variation(Variation shorter) {
-			this.shorter = shorter;
+		private Variation(String translateKeySuffix) {
+			this.translateKeySuffix = translateKeySuffix;
+		}
+		
+		public TranslatableText getTranslatableText(String translationKey) {
+			return new TranslatableText(translationKey + translateKeySuffix);
+		}
+		
+		public Variation getNextVariation(int amount) {
+			int targetOrdinal = this.ordinal() + amount;
+			if(targetOrdinal < 0 || targetOrdinal >= Variation.values().length) {
+				return null;
+			}
+			return Variation.values()[targetOrdinal];
```

Comment:
Probably not. But at this point I was more concered about useability. This allows walking the values up and down rather than only one direction.

Reply:
I don't see myself walking the values in the other direction (and it would still be possible through the normal enum methods) but I'm gonna accept that.